docs/app/(docs)/commands/ballot/page.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Command: `ballot` - SudoBot"
+short_name: ballot
+---
+
+import Modes from "@/components/MDX/Modes";
+
+# `ballot`
+
+Turns anti join on or off. Makes it so that if someone joins while it's activated, they get kicked from the server.
```

Comment:
Pretty sure this is a mistake lol. Would you mind correcting this, or I can make a patch if you want!

Reply:
That was a mistake lol, I'll correct it now, one minute.

Reply:
Fixed!